### DISCUSSION TITLE
longhorn log size  too too too big , how to solve it

### DISCUSSION THREAD

**Discussion by zbjjyy** [longhorn/longhorn#5890] [General]
the DeamonSet ,longhorn-manager, print larges of logs,how should i change it?

where i `echo""` to the json file ,the it's size change again 
`1.5T    ece119dd4a667c12180dac53e658d5a955a077295f227b174c3af675be8848c3`
```
root@sedtwork2:/mnt/dockerdata/containers/ece119dd4a667c12180dac53e658d5a955a077295f227b174c3af675be8848c3# ll
total 14682268
drwx--x---    4 root root        4096 5月   6 09:31 ./
drwx--x--- 1081 root root      233472 5月   6 09:48 ../
drwx------    2 root root        4096 5月   4 12:20 checkpoints/
-rw-------    1 root root       12630 5月   4 12:20 config.v2.json
-rw-r-----    1 root root 15034359875 5月   6 09:49 ece119dd4a667c12180dac53e658d5a955a077295f227b174c3af675be8848c3-json.log
-rw-r--r--    1 root root           1 5月   6 09:31 ece119dd4a667c12180dac53e658d5a955a077295f227b174c3af675be8848c3-json.logece119dd4a667c12180dac53e658d5a955a077295f227b174c3af675be8848c3-json.log
-rw-r--r--    1 root root        2062 5月   4 12:20 hostconfig.json
drwx--x---    2 root root        4096 5月   4 12:20 mounts/
```

**Comment by innobead** [comment]
The only way, for now, should be container runtime for log rotation. It seems you are using docker. probably need to check the related setting. 

Some improvements will be made in https://github.com/longhorn/longhorn/issues/5888

**Reply by zbjjyy** [reply]
yes, container runtime is docker.and the log size changed so large due to a rezing pvc````
i have fixed the promble ,3q
